tests/selenium/start.js

```diff
@@ -5,64 +5,16 @@ var driver = new webdriver.Builder().
   withCapabilities(webdriver.Capabilities.firefox()).
   build();
 
+var testRunner = require('./test-runner');
+var sequentialForm = require('./sequential-form-test.js')(driver, By);
```

Comment:
EH, just trying it with one less thing to remember to require in for each test file.